package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ember-ghost-w3css-blog-template",
+  "name": "empress-blog-w3css-blog-template",
   "version": "0.1.1",
   "description": "This is an implementation of the W3CSS Blog Template for Empress Blog",
   "keywords": [
```

Comment:
👇 you probably want to change the repo name in package.json too, but then you would want to update the actual repo name by renaming it in github 😂